setuptools_rust/setuptools_ext.py

```diff
@@ -162,6 +162,7 @@ class bdist_wheel_rust_extension(bdist_wheel_base_class):
             def finalize_options(self):
                 scripts = []
                 for ext in self.distribution.rust_extensions:
+                    ext.py_limited_api = self.py_limited_api
```

Comment:
Hmm I'd rather we didn't modify the `RustExtension` objects. I think we can do this an alternative way by loading the `bdist_wheel` command when we want to decide whether to build a limited api wheel.

Reply:
(fixed in my PR)